jaq-core/src/lib.rs

```diff
@@ -402,3 +404,82 @@ const LOG: &[(&str, usize, RunPtr, UpdatePtr)] = &[(
     |_, cv| box_once(Ok(debug(cv.1))),
     |_, cv, f| f(debug(cv.1)),
 )];
+
+#[cfg(feature = "format")]
+fn stringify(v: Val) -> String {
+    v.clone()
+        .to_str()
+        .map_or_else(|_| v.to_string(), |s| s.to_string())
```

Comment:
I would replace s.to_string() by  something like s.clone()  here, so it is clear that we are really just reusing the string that is there and not creating an escaped version of it.